packages/remix-docs-gen/src/routes.ts

```diff
@@ -43,7 +43,24 @@ export const parseRoutes = (regexFilter?: string) =>
           )
           .filter((item) => Boolean(item.path) && !item.path.includes("/*"));
 
-      let routes = parseRoutes(JSON.parse(output));
+      let foundRoutes = parseRoutes(JSON.parse(output));
```

Comment:
I just noticed that the way you're algorithm currently works it doesn't pick up on the root loader, so I figured "Why the heck not?". If you would like this in a seperate PR I'd be happy to split it apart.